src/Emulator/Peripherals/Peripherals/Mocks/HPSHostController.cs

```diff
@@ -180,6 +187,12 @@ public byte[] ReadMemoryBankAvailable(TimeInterval timeInterval)
             return FormatCommonErrorStatus(GetBytesFromSlave(2, timeInterval));
         }
 
+        public byte[] ReadErrorBytes(TimeInterval timeInterval)
```

Comment:
I suggest refactring `ReadError` to:
```
        public string[,] ReadError(TimeInterval timeInterval)
        {
            return FormatCommonErrorStatus(ReadErrorBytes(timeInterval));
        }
```
to avoid code duplication.

Reply:
Sounds good.